step4/api.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express')
+const router = express.Router();
+const fs = require('fs');
+const { v1: uuidv1 } = require('uuid'); // 고유 id를 생성해주는 uuid 사용
+const {body, validationResult} = require('express-validator');
+
+
+// const jsonFile = fs.readFileSync('./data.json', 'utf-8'); // json string 그대로의 파일
+// console.log(jsonFile)
+// const items = JSON.parse(jsonFile).items // json을 object type으로 변환 
+// console.log(items)
+
+// const ID = uuidv1(); // 고유 id 생성
+// console.log(ID)
+
+/**
+ * 아이템 조회
+ */
+router.get('/items', (req, res, next) => {
+    const data = JSON.parse(fs.readFileSync('./data.json', 'utf-8', (err) => {
+        if(err) next(err)
+    }))
+
+    res.status(200).json({items: data.items, selectedItem: data.selectedItem});
+    
+})

```

Comment:
```suggestion
router.get('/items', (req, res, next) => {
    try {
      const data = JSON.parse(fs.readFileSync('./data.json', 'utf-8');
      res.status(200).json(data);
    } catch(err) {
      next(err)
    }
})
```
이렇게 try/catch로 표현해줘야 할 것 같아요!